### ISSUE TITLE
Error on Google extention with manifest v3

### ISSUE THREAD

**Issue by young-healer** [cosmos/cosmjs#964]
Hi,
I face a problem on developing Google extention on manifest v3 with this library,
After build and deploy i got this error, and can't find a way to fix this
"Uncaught (in promise) RuntimeError: abort(CompileError: WebAssembly.instantiate(): Wasm code generation disallowed by embedder)"

**Comment by webmaster128** [comment]
Right now, CosmJS uses Wasm internally for crypto. We may migrate away from that at some point, but cannot easily do that short term. Did you see this? https://stackoverflow.com/questions/48523118/wasm-module-compile-error-in-chrome-extension

**Comment by young-healer** [comment]
@webmaster128 Yes, I've read that before. these methods don't work with the third manifest.

**Comment by webmaster128** [comment]
This is discussed here: https://bugs.chromium.org/p/chromium/issues/detail?id=1173354 I do not yet understand all the details and have no immediate solution.

**Comment by ducciobottai** [comment]
Are there any news about this issue? We're developing a Chrome extension and this is a huge blocking issue. Please let me know!

**Comment by webmaster128** [comment]
Contributions are welcome, especially researching what the core problem is here and what the library can do. I'll not work on this myself any time soon.

**Comment by dynst** [comment]
The problem is specifically a dependency on a WebAssembly build of libsodium, right? Which can be replaced with a pure JavaScript dependency.
